document/src/main/java/com/ritense/document/config/SpringContextHelper.java

```diff
@@ -33,4 +35,9 @@ public void setApplicationContext(@NotNull ApplicationContext applicationContext
     public static <T> T getProperty(String property, Class<T> targetClass) {
         return context.getEnvironment().getProperty(property, targetClass);
     }
+
+    public static <T> Map<String, T> getBeansOfType(Class<T> targetClass) {
```

Comment:
Generally speaking this is a security risk, as now there's a static method that can retrieve any bean at every point in the code. I suggest making a package private variation of this SpringContextHelper that only retrieves FormFieldDataResolver beans.